### DISCUSSION TITLE
RDB 는 언제 쓰고 NoSQL은 언제 쓰나요?

### DISCUSSION THREAD

**Discussion by leehosung** [8percent/rdb-study#10] [Q&A]
#1
알려줘요

**Comment by wer153** [comment]
RDBMS
- 데이터 구조가 명확하여 스키마의 변경이 빈번하지 않을 경우.
- update빈도가 높을 경우
- 무결성이 중요한 경우.

NoSQL
- 데이터 구조의 변경 확장이 잦을 경우.
- write빈도가 높을경우.
- scale-out이 필요한 경우. (1억 레코드 이상)

**Reply by kyungjunleeme** [reply]
![image](https://user-images.githubusercontent.com/45473846/145510285-250ca1d1-583d-4efb-89f3-8f483eee4aec.png)
출처 : https://meetup.toast.com/posts/274
[요약] mongoDB를 통해 NoSQL의 특징을 살펴보면서, 관계형 DB와의 비교 내용이 자세하게 기술되어 있습니다.
